certificates_transferring_certificates.md

```diff
@@ -10,43 +10,43 @@ This help document will walk through how to transfer certificates
 
 <br>
 
-## How to Transfer Certificates?
+## How to Transfer Certificates:
 
-Navigate to the Certificates Dashboard: Active Tab. Select the batch(es) of Certificates you wish to transfer. Once you have selected the batches select the "Transfer" button. 
+Navigate to the Certificates Dashboard: Active Tab. Select the batch(es) of Certificates you wish to transfer. Once you have selected the batches select the 'Transfer' button. 
 
 ![](https://github.com/markmrets/photos/blob/master/transfering%20certs-%20step%201.png?raw=true)
 
 <br>
 
-If you would like to change the number of certificates you are transferring, you can do so here by entering the desired quantity in the "Quantity" field. After doing so select "Next".
+If you would like to change the number of certificates you are transferring, you can do so here by entering the desired quantity in the "Quantity" field. After doing so select 'Next'.
```

Comment:
I think "Next" is correct. I think you should only use single quotation marks when stating a quote within a quote. I could be wrong.